src/main.cc

```diff
@@ -67,12 +68,58 @@ void process_event(const std::shared_ptr<event::BaseEvent> &event) {
               << (install_complete->success ? "success" : "failure") << "\n";
   } else if (event->variant == "UpdateCheckComplete") {
     const auto check_complete = dynamic_cast<event::UpdateCheckComplete *>(event.get());
-    std::cout << check_complete->result.updates.size() << " updates available\n";
+    switch (check_complete->result.status) {
+    case result::UpdateStatus::kError:
+      std::cout << check_complete->result.updates.size() << " updates with error";
+      break;
+    case result::UpdateStatus::kUpdatesAvailable:
+      std::cout << check_complete->result.updates.size() << " updates available";
+      break;
+    case result::UpdateStatus::kNoUpdatesAvailable:
+      std::cout << check_complete->result.updates.size() << " updates unavailable";
```

Comment:
This will read `0 updates unavailable`, which is grammatically confusing. A simple "No updates available" would be better, but is there not already an appropriate log message printed from libaktualizr?

Reply:
Good point, will address in next revision. There's log from aktualizr but it would be nice to have independent message in case we're disabling aktualizr logging by setting log-level - to decrease log noise for example.

Reply:
That's a fair point about running with a loglevel that doesn't print that. Currently the equivalent message in libaktualizr is at debug level. That means this is probably fine, although we might want to consider only printing it if the loglevel is greater than debug.

The grammatical error needs to be fixed in any case, though.